README.md

```diff
@@ -124,13 +124,13 @@ Key:
 * `✓` Changes in main Kubernetes repo are manually ([should be automated](https://github.com/kubernetes-client/python/issues/177)) published to client-python when they are available.
 * `✗` No longer maintained; please upgrade.
 
-Kubernetes supports [three minor releases](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) at a time. "Support" means we expect users to be running that version in production, though we may not port fixes back before the latest minor version. For example, when v1.3 comes out, v1.0 will no longer be supported. In consistent with Kubernetes support policy, we expect to support **three GA major releases** (corresponding to three Kubernetes minor releases) at a time.
+Kubernetes supports [three minor releases](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/release/versioning.md#supported-releases-and-component-skew) at a time. "Support" means we expect users to be running that version in production, though we may not port fixes back before the latest minor version. For example, when v1.3 comes out, v1.0 will no longer be supported. In consistent with the Kubernetes support policy, we expect to support **three GA major releases** (corresponding to three Kubernetes minor releases) at a time.
 
-Note: There would be no maintenance for alpha/beta releases except the latest one.
+Note: There would be no maintenance for alpha/beta releases except for the latest one.
```

Comment:
Except and except for are often considered the same given that the phrase is followed by a noun. See [the grammar post here](https://dictionary.cambridge.org/us/grammar/british-grammar/except-or-except-for). In this case, the correct meaning is "there would be no maintenance in addition to the last one", so the word "besides" is more accurate.

Reply:
revision done. waiting for further feedback.